### PR TITLE
duckdb: add version 0.8.1

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **duckdb/0.8.1**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/duckdb/all/patches/0.8.1-0001-fix-cmake.patch [hunk 1]
```diff
@@ -0,0 +1,87 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
```
Comment (memsharded) [inline recipes/duckdb/all/patches/0.8.1-0001-fix-cmake.patch:1]: It seems this patch is the same as the one for 0.8.0? I fail to see any difference

Comment (toge) [reply recipes/duckdb/all/patches/0.8.1-0001-fix-cmake.patch:1]: @memsharded 
line numbers in tools/sqlite3_api_wrapper/CMakeLists.txt are different.

in 0.8.0
```
--- a/tools/sqlite3_api_wrapper/CMakeLists.txt
+++ b/tools/sqlite3_api_wrapper/CMakeLists.txt
@@ -19,7 +19,7 @@ if(NOT AMALGAMATION_BUILD)
 endif()
 link_threads(sqlite3_api_wrapper_static)
```

in 0.8.1
```
--- a/tools/sqlite3_api_wrapper/CMakeLists.txt
+++ b/tools/sqlite3_api_wrapper/CMakeLists.txt
@@ -26,7 +26,7 @@ if(NOT AMALGAMATION_BUILD)
 endif()
 link_threads(sqlite3_api_wrapper_static)
```

**Review by RubenRBS — APPROVED** [review]

File: recipes/duckdb/all/patches/0.8.1-0001-fix-cmake.patch (hunk 1 above)
Comment (memsharded) [reply recipes/duckdb/all/patches/0.8.1-0001-fix-cmake.patch:1]: Thanks for the clarification, I didn't realize this while reviewing!

**Review by memsharded — APPROVED** [review]

**Comment by SpaceIm** [comment]
One patch shouldn't be there in 0.8.1:

```
duckdb/0.8.1: Apply patch (portability): include stdlib for abort function
duckdb/0.8.1: WARN: /home/spaceim/.conan2/p/duckdda50c49d15e7e/s/patches/0.6.0-0002-include-stdlib.patch: file 1/1:         b'extension/jemalloc/jemalloc/include/jemalloc/internal/jemalloc_preamble.h'
duckdb/0.8.1: WARN: /home/spaceim/.conan2/p/duckdda50c49d15e7e/s/patches/0.6.0-0002-include-stdlib.patch:  hunk no.1 doesn't match source file at line 7
duckdb/0.8.1: WARN: /home/spaceim/.conan2/p/duckdda50c49d15e7e/s/patches/0.6.0-0002-include-stdlib.patch:   expected: b''
duckdb/0.8.1: WARN: /home/spaceim/.conan2/p/duckdda50c49d15e7e/s/patches/0.6.0-0002-include-stdlib.patch:   actual  : b'#include <cstdlib>'
duckdb/0.8.1: WARN: /home/spaceim/.conan2/p/duckdda50c49d15e7e/s/patches/0.6.0-0002-include-stdlib.patch: already patched  b'extension/jemalloc/jemalloc/include/jemalloc/internal/jemalloc_preamble.h'
```

**Pull request merged**